samples/python/pyproject.toml

```diff
@@ -3,7 +3,7 @@ name = "a2a-samples"
 version = "0.1.0"
 description = "Agent2Agent samples"
 readme = "README.md"
-requires-python = ">=3.12"
+requires-python = ">=3.11"
```

Comment:
I didn't find any dependency that requires Py 3.13 (docs show Py 3.13 as required version, however this config allows for 3.12). One can install uv with Py 3.11/3.12 by running `uv venv --python "3.11"` or `uv venv --python "3.12"`.